container/new_pull.bzl

```diff
@@ -0,0 +1,209 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
I think you probably don't want the starlark part in this PR?

Reply:
git problem again. I switched branches but it somehow didn't push properly  :(

Reply:
I'll delete the file and try again.